dom.bs

```diff
@@ -1028,6 +1028,18 @@ specified otherwise it returns null.
 <p class="note no-backref"><a>Nodes</a>, <a for=/>shadow roots</a>, and <a>documents</a> override
 the <a>get the parent</a> algorithm.
 
+<p>Each {{EventTarget}} object can have an associated
+<dfn export for=EventTarget>legacy pre-click-dispatch steps</dfn> algorithm,
+<dfn export for=EventTarget>legacy post-canceled-click-dispatch steps</dfn> algorithm, and
+<dfn export for=EventTarget>activation behavior</dfn> algorithm. The <a>activation behavior</a>
+algorithm is passed an <a>event</a>, as indicated in the <a>dispatch</a> algorithm.</p>
+
+<p class="note no-backref">An {{EventTarget}} object that has <a>legacy pre-click-dispatch steps</a>
```

Comment:
Yeah, I should rephrase it as saying that you can have activation behavior and that if you have that, you can additionally have both pre-click and post-canceled-click. That would be a lot more clear and not warrant a note to state the obvious.